invenio_rdm_migrator/load/postgresql/table_generators/mapper.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Migrator is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Table generator mapper module."""
+
+from ....logging import Logger
+
+
+class TableGeneratorMapper:
```

Comment:
The actual concept could be understood as demultiplexing.
```suggestion
class TableGeneratorDMUX:
```